helpers/getAllPosters.js

```diff
@@ -0,0 +1,18 @@
+import { MongoClient } from "mongodb";
+
+export async function getAllMoviePosters() {
+
+
+    const client = await MongoClient.connect("mongodb+srv://Bosse:LKjRPJ2chOlxeM0E@cluster0.rpxxl.mongodb.net/moviePosters?retryWrites=true&w=majority");
```

Comment:
You should not include this type of information in the code. It means you are leaking database credentials that anyone (with access to your repo, which is anyone because it's a public repo) can see and steal those credentials.

Consider using environment variables for information like this. And remember that if it was ever pushed to the repo (like here) it remains in the history even if you delete it from the file. So after accidentally pushing this type of information it's important to change the credentials (e.g. the password) on the database server.